### DISCUSSION TITLE
v23.2

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#3296] [General]
**The installers are located in the "Assets" section at the end of these release notes!**

[![Github Release](https://img.shields.io/github/downloads/Stellarium/stellarium/v23.2/total.svg)](https://github.com/Stellarium/stellarium/releases/v23.2)

## Release notes
The Stellarium Team has released version 23.2.

The major changes of this version:
- vectorize marker of deep-sky objects
- updated support locations
- discovery circumstances for solar system objects

Behind the scenes, many more minor issues were fixed.

TELESCOPE USERS

Like in the 1.* series, it seems the TelescopeControl plugin at least on Windows works better in the Qt5-based releases. Just install this if you are affected. We are looking for telescope users with programming skills to help us fixing this issue. Reward possible!

## What's Changed
Full list of changes:
- Added ecliptic obliquity to infoMap (GH: #3233)
- Added support "short-period comets" type
- Added support combined designations for numbered periodic comets
- Added optional "Solar A/a" and "Lunar A/a" info for objects (GH: #3273)
- Added support for webp format in HiPS
- Added discovery circumstances for Caldwell objects
- Added discovery circumstances for Solar System Objects (GH: #3230)
- Added support antialiasing for Oculars lines
- Added changeable font to LeftStelBar (GH:  #3125)
- Added UTM/WGS84 coordinate conversions
- Added push to directional light strength in Scenery 3D plugin
- Added new category into AstroCalc/Phenomena tool
- Added missing timezone (GH: #3200)
- Added push to directional light strength into Scenery 3D plugin
- Added new rotation buttons for CCD in the Oculars plugin
- Added KY day number in Calendars plugin
- Added option "Stop clock when starting" (GH: #3140)
- Added ability to specify seconds in time selectors of AstroCalc/Ephemeris tool (GH: #3146)
- Fixed building Satellites plugin when Iridium flares are enabled
- Fixed 64-bit install mode for Windows (GH: #3130)
- Fixed angular size for AstroCalc/Graphs tool: Let's use angular radii of planets without rings (GH: #3135)
- Fixed computation of perpendicular in wide line rendering
- Fixed tooltips for Penumbra and Umbra circles (GH: #3175)
- Fixed updating location even if name is the same (GH: #3171)
- Fixed fading of the new nebula markers
- Fixed Observing Lists feature: avoid duplicate stored list when creating a new list (GH: #3163)
- Fixed missing doSRGB uniform in AtmosphereShowMySky
- Fixed filters in Satellites plugin (GH: #3158)
- Fixed checking updates
- Fixed working the actions in Oculars plugin (GH: #3280)
- Fixed prism rotation (GH: #3283)
- Fixed links to Mike McCants' website
- Fixed designations of periodic comets
- Fixed WUI of Remote Control plugin
- Fixed orbital elements of Phoebe (GH: #230)
- Fixed special handling of Earth map (GH: #3180)
- Fixed a bad connection definition (GH: #3180)
- Fixed missing include (GH: #3180)
- Fixed orbital elements of Pluto's moons (GH: #3242)
- Fixed missing data for periodic comets
- Fixed label for button in Equation of Time plugin
- Fixed Greek (Farnese) sky culture
- Fixed positioning of sensor labels with high-DPI scaling on (GH: #3262)
- Fixed orientation of sensor rectangle in equatorial mode (GH: #3260)
- Fixed missing rotation of sensor frame (GH: #3258)
- Fixed high-DPI-scaled mouse click position test relative to Oculars panel
- Fixed computation of inner and outer radii of OAG
- Fixed shapes of ocular frames
- Fixed computation of angular resolution in Oculars plugin
- Fixed positioning of text around the sensor frame in Oculars plugin (GH: #3254)
- Fixed a little mess with loading landscapes
- Fixed metainfo for AppData scheme
- Fixed rose32 directions (GH:  #3235)
- Fixed a "round cap near zenith" artifact of ShowMySky atmosphere
- Fixed checking updates
- Fixed skewed text in -t mode
- Fixed text element arrangement in buttonbar (GH:  #3125)
- Fixed template string for width estimate (GH:  #3125)
- Fixed a possible glitch when too few buttons are visible (GH:  #3125)
- Fixed bug in AstroCalc/RTS tool
- Changed core: slight improvement of gravity labels
- Changed core: Suppress computing RTS when on transition to a new location (GH: #3180)
- Changed core: Avoid computing minor bodies while on a transitioning observer (GH: #3180)
- Changed core: When autoselecting landscape and planet landscape not found, don't default to defaultLandscape but use zero (GH: #3180)
- Changed core: Load zero landscape if landscape not found (GH: #3180)
- Changed core: Unselect object when it is the upcoming location's planet (GH: #3180)
- Changed core: When clicking on the map, apply a zero landscape with color from the map (GH: #3180)
- Changed core: Allow dynamic loading of colorized zero landscape everywhere (GH: #3180)
- Changed core: Avoid auto-enabling atmosphere if we stay on the same planet (GH: #3180)
- Changed core: Use ZeroColor landscape also only when "Autoselect Landscapes" (GH: #3180)
- Changed core: Use actual planet texture name (GH: #3180)
- Changed core: make sure AtmosphereShowMySky is initialized with correct altitude
- Changed core: deduplicate rendering of horizon line
- Changed core: render fixed-color wide lines via triangles (GH: #3187, #2836)
- Changed core: specify texture unit explicitly in StelPainter::drawText (GH: #3154)
- Changed core: a small speed-up for DSO
- Changed core: reduce number of texture loading threads for most systems (GH: #3148)
- Changed core: simplify setting OpenGL version
- Changed core: make sure we do request deprecated functions in Compatibility Profile
- Changed core: vectorize marker of galaxy (GH: #3188)
- Changed core: vectorize marker of globular cluster and the default circular marker
- Changed core: vectorize marker of open cluster with nebulosity (GH: #3172)
- Changed core: vectorize marker of open cluster (GH: #3172)
- Changed core: vectorize markers of dark nebulas (GH: #3166)
- Changed core: vectorize markers of diffuse nebulas (GH: #3166)
- Changed core: allow SOCKS proxy to be supplied via http_proxy env var (GH: #3182)
- Changed core: Make linearRGB/sRGB conversion GLSL functions reusable
- Changed core: Make rectangular FoV marking follow real projection (GH: #3290)
- Changed GUI: reduce margins of DateTime dialog's spinbox buttons
- Changed GUI: Reduce margins of DateTime dialog's spinbox buttons (GH: #2722, #3157)
- Changed GUI: Use separate spinup/down icons for DateTime dialog spinboxes (GH: #2722, #3157)
- Changed GUI behavior: force GUI redraw when topocentric correction is toggled (GH:  #3125)
- Changed GUI behavior: improve font-dependent positioning of bottom button bar (GH:  #3125)
- Changed GUI behavior: better distance distribution of status line elements (GH:  #3125)
- Changed GUI behavior: improve positioning of texture-based text
- Changed GUI in AstroCalc/Positions tool: let's use arcseconds for SSO
- Changed Scenery3D plugin: allow ignoring initial view settings
- Changed Scenery 3D plugin: Allow coordinate text output in bottom/center (fix for planetarium)
- Changed Satellites plugin: allow Satellite Number to be zero (GH: #3186)
- Changed AstroCalc tool: when switching to best locations, use zero landscape
- Changed documentation: improve Fedora building instructions (GH: #3192)
- Changed AppImage properties: Disable using QtWebEngine for AppImage (GH: #3143)
- Updated labels in AstroCalc
- Updated list of financial supporters
- Updated data: Revising default list of comets (GH: #3276)
- Updated list of locations
- Updated discovery circumstances for minor planets
- Updated documentation: Specify tab size explicitly in coding style document (GH: #3284)
- Updated list of contributors
- Updated CalcMySky to v0.3.1
- Updated QXlsx to v1.4.6 (GH: #3132)
- Updated translations
- Updated data: revised discovery circumstances of Messier objects
- Updated common names of DSO
- Updated Stellarium User Guide
- Updated planetary features
- Updated catalog of exoplanets
- Updated orbital elements for minor bodies of Solar system
- Removed: Text User Interface plugin has been disabled by default now
- Removed separate color for clusters with nebulosity (GH: #3172)

## Packages notes
package | note
------------|---------
stellarium-23.2.tar.gz | Stellarium 23.2 for UNIX/Linux (source code). <br />__MD5:__ b42a829ffe080ba7c27b8ff25ba8a9c8 <br />__SHA256:__ 01bf23f7ff007c58b91bffe233938e2cf23e9182bd36bfe72715fbfbde57ec0a
stellarium-23.2.tar.bz2 | Stellarium 23.2 for UNIX/Linux (source code). <br />__MD5:__ 7ec2ae76d45e902ac6c94db6e2b6214c <br />__SHA256:__ 191e37bf1d8528c90d87d66f204fde79c8580f9f6d180aefffede57823ada611
stellarium-23.2.tar.xz | Stellarium 23.2 for UNIX/Linux (source code). <br />__MD5:__ b67149a2220752978a58a244d5b933ae <br />__SHA256:__ 3aee154d0de3ed484e3e6fee027cf73e6dd0a677b3ea1af348b225ac02339008
stellarium-23.2-qt5-win32.exe | Stellarium 23.2 for Windows (32-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: f6edfc098c5fbe4fa33f4bfcfb06ab33 <br />__SHA256__: 49d41ea0606e544fb4063edd11af0d82c68f77c41111116c69e2293d1be9d0bb
stellarium-23.2-qt5-win64.exe | Stellarium 23.2 for Windows (64-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: a41fc9e226814ef88d19768968619905 <br />__SHA256__: 3eddece0d30e3a6a7a1ff8d34802936ef97e62ecba625fd9339260731cce4c7f
stellarium-23.2-qt6-win64.exe | Stellarium 23.2 for Windows (64-bit; Windows 10+) based on Qt 6.4.2 (Visual Studio 2019 Community) with media support. <br /> __MD5__: 2437899d9f132f6d898aac3af91bb47a <br />__SHA256__: faf7913c9ff87d5844d2fbe61f97f56da8161b4e2d6932fadc52d0da318222a2
Stellarium-23.2-qt6-macOS.zip | Stellarium 23.2 for macOS (universal; macOS 11.0+) based on Qt 6.5.1 (AppleClang 14.0.0) with media support. <br /> __MD5__: 6205fb22cefc2623bf6ec7e17c2ce2ce <br />__SHA256__: 1cddac3c5aeedde5012a2e3696fb76a432ed4da323f702ce646c82f90dfee335
Stellarium-23.2-qt5-arm64.zip | Stellarium 23.2 for macOS (arm64; macOS 10.14+) based on Qt 5.15.12 (AppleClang 14.0.0) with media support. <br /> __MD5__: 9cfe811c08bb72ef555d3a28f0541f3b <br />__SHA256__: 1b01464ce04d75ca1c9d0ddfdf776f1afb6b4c160a0a5c723da79bcf4e82bbab
Stellarium-23.2-qt5-x86_64.zip | Stellarium 23.2 for macOS (x86_64; macOS 10.14+) based on Qt 5.12.8 (AppleClang 11.0.0) with media support. <br /> __MD5__: da9703ce4607f92d4a3341318800d555 <br />__SHA256__: 44b9c9ceac4c6f2bb4a6ab722358794d9d7163c06ff4c9bfe4afb48292b28152
Stellarium-23.2-qt5-x86_64.AppImage | Stellarium 23.2 for Linux (amd64; glibc 2.31) based on Qt 5.12.8 (GCC 9.4.0) with media support. <br /> __MD5__: c0a82620a1a6129a9a570d9acca64869 <br />__SHA256__: 5ccfc0b3db277f3f8d3d43a8fb92fc5c239fd87ed71485a1d579cf796256ae10

### GPG signature
1. Import Stellarium's GPG key
  `curl -O https://stellarium.org/files/stellarium.gpg`
2. List Stellarium's GPG key
  `gpg --with-fingerprint --show-keys --keyid-format long stellarium.gpg`
  **id:** `rsa4096/BF38D4D02A328DFF 2012-04-07`
  **fingerprint:** `7915 1C2E 6351 E727 8DA1  A730 BF38 D4D0 2A32 8DFF`
3. Verify the binary file is valid (tarball for example)
  `gpgv --keyring ./stellarium.gpg stellarium-23.2.tar.gz.asc stellarium-23.2.tar.gz`

<hr /><em>This discussion was created from the release <a href='https://github.com/Stellarium/stellarium/releases/tag/v23.2'>v23.2</a>.</em>

**Comment by Suby81** [comment]
It appears that stellarium-23.2-qt5-win64.exe is missing from the released files

**Reply by alex-w** [reply]
qt5-based packages for Windows will be publish tomorrow

**Comment by tajicoji** [comment]
I installed ver23.2 on Windows 11, connected the mount and tested it on my desk, but after moving a little, it suddenly quits. 
This happens many times. 
There are no errors in the logs. I attach the log.
[log.txt](https://github.com/Stellarium/stellarium/files/11933800/log.txt)

**Reply by alex-w** [reply]
Please try qt5-based package

**Reply by tajicoji** [reply]
Thank you! It's working without errors!

**Reply by tajicoji** [reply]
I currently have the latest version of qt5 installed, but when I try to connect from APT, I get the following error and can't connect.
The APT forum says it's a Stellarium issue, what should I do?
(qt6 is NG due to frequent crashes)

OpErr	Connecting to Stellarium problem (5)
OpErr	Charset must be iso-8859-1, utf-8, utf-16, utf-16le, or utf-16be to be extracted.

**Reply by alex-w** [reply]
Yes, this is a problem in Stellarium x64. Sorry, we have no solution yet, but we're working on this issue.

**Reply by tajicoji** [reply]
Is it an x64 issue? stellarium-0.22.0-win64.exe works fine.

**Reply by alex-w** [reply]
stellarium-0.22.0-win64.exe is Qt5-based and package was produced before changes code to compatible with Qt6 and Qt5.15

**Reply by alex-w** [reply]
Ouch... APT can connect to Stellarium x64/Qt6 now for master O_O

I'll publish development snapshot in this weekend and I hope you can test it

**Reply by tajicoji** [reply]
>Ouch... APT can connect to Stellarium x64/Qt6 now for master O_O

What does 'master O_O' mean?
Are you using a non-major release version of APT?

>I'll publish development snapshot in this weekend and I hope you can test it

Will you provide a test version of Stellarium over the weekend?

**Reply by alex-w** [reply]
master = Stellarium v23.2.186

> Will you provide a test version of Stellarium over the weekend?

Yes

**Reply by tajicoji** [reply]
oh! I mean, maybe it went well. Looking forward to the weekend!

I have downloaded various versions.
stellarium-23.2-qt6-win64>> Frequent crashes
stellarium-23.2-qt5-win64 >>> Doesn't crash, but can't connect to remote control from APT
stellarium-23.1-qt5-win64>> Same as ↑
stellarium-1.2-qt5-win64>> Same as ↑
stellarium-0.22.1-win64 >>　The remote control connection is OK, but if the window is made full size, it will black out when closing or switching to another window (about 1 second)
stellarium-0.22.0-win64 >> Same as ↑

PC environment
OS: Windows 11 Pro 22H2 64bit
PC: Intel(R) Core(TM) i3-1005G1 CPU @ 1.20GHz

**Reply by tajicoji** [reply]
By the way, I think the coordinates are off.
This is a Capella, but the J2000 says (RA) 05h 16m 41.35871, (DEC) +45° 59' 52.7693″, but the Stellarium says (RA) 05h 16m 39.89, (DEC) +45° 59' 38.0 .

![capella](https://github.com/Stellarium/stellarium/assets/84647917/96843166-48d3-4da8-a47c-33c420611aaf)

**Comment by alex-w** [comment]
Hello @tajicoji!

Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by tajicoji** [reply]
Hi, Alex

The result of the test immediately is regrettable, but it is NG.
”stellarium-23.2-a69d242-qt6-win64.exe”
I was able to connect to remote control from APT, but it still crashes suddenly, so I can't use it.

”stellarium-23.2-a69d242-qt5-win64.exe”
Remote control connection from APT is not possible.
APT still has the same message.
In addition, the drawing of the ground surface is supposed to be outrageous. (Image attached)
![WS000042](https://github.com/Stellarium/stellarium/assets/84647917/ce3eb17b-5ca2-4810-a77b-60f7a85da373)

**Reply by gzotti** [reply]
> APT still has the same message.

Which is: ... ?

The ground looks like Zero horizon. It is hard to say what is outrageous about it, what did you expect to see? Please read changes in the User Guide, ch.4.2.

**Reply by tajicoji** [reply]
I'm sorry for this. Unlike older versions, the default landscape is not set. understood.
However, in any case, snapshot qt6 crashes even if you can connect remotely, and qt5 cannot be used because it has not changed without being able to connect remotely.

**Reply by tajicoji** [reply]
Here are the test results after that.
I am using stellarium-23.2-a69d242-qt6-win64.
This build works fine for remote connections from APT, but crashes frequently. However, the crash only seems to occur when connecting via telescope control from Stellarium.
Maybe the cause of the crash is a problem with the telescope control.
The telescope has been tested with the following drivers from ASCOM and they all crashed.
  Simulator
  Telescope Simulator for .NET
  Gemini Telescope .NET

**Reply by alex-w** [reply]
Please fill bug report and attach the log.txt

**Reply by gzotti** [reply]
Crashes with qt6-based TC have been reported, e.g. #2821.

**Comment by Piekarnik** [comment]
New installed. I want to see ISS. Stellarium shows my time (taken from computer) and it's correct. Also correct is sky colour. Stellarium shows ISS is on my sky at current time. I run outdoor and there is not. ISS arrives one hour later when the sky is dark. I think Stellarium counts something wrong. Could it check online that something isn't right?

**Reply by gzotti** [reply]
Is your location correct?  Have you updated satellites (TLE) lately?

**Reply by Piekarnik** [reply]
TLE updated. My location is absolutely correct. The problem is: Stellarium showed ISS one hour too early. I've been checking ISS positon on https://spotthestation.nasa.gov/tracking_map.cfm and ISS was just over the Pacific Ocean. ISS arrived exactly one hour later than Stellarium showed.

**Reply by alex-w** [reply]
Sorry, but how we can help you if you didn't give us any technical info? OK, you say it's a newly installed instance of Stellarium and your location is correct - which location and what time zone is set? You say TLE updated - OK, when you select ISS, then what you see for "Last updated TLE" and "Epoch TLE" items in info block?

**Reply by gzotti** [reply]
Is your object named "ISS (Zarya)"?

**Reply by Piekarnik** [reply]
I've been trying to reload data about satelites. Then checked ISS position. Now it's correct. Thank you very much for an effort!

**Comment by 72and24** [comment]
I've searched through the Stellarium 23.2 Configuration tabs and Sky and Viewing Options tabs; but ... 
I can't find how/where to turn off the Calendar Info that's blocking part of the sky. (Gregorian / Julian / ISO Week / Islamic / Hebrew / ...) 
Any help?

**Reply by 72and24** [reply]
Doing a word search for 'calendar' in the User Guide lead me to (Hotkeys Appendix) ... Alt + K to toggle the display. 

But, I'm accustomed to being able to find such options in the Configuration or Sky & Viewing. 
Still haven't found the option there. 

P.S.: Found that Shift + M will toggle the Meteor Shower info / helps when trying to achieve a naked-eye view of the sky.

**Reply by gzotti** [reply]
What you are accustomed to may not be what developers have created. Plugins have their own setting GUIs.  To clarify, there is the User Guide.
